packages/stargate/src/logs.ts

```diff
@@ -69,9 +69,8 @@ export function parseRawLog(input = "[]"): readonly Log[] {
  *
  * Throws if the attribute was not found.
  */
-export function findAttribute(logs: readonly Log[], eventType: string, attrKey: string): Attribute {
-  const firstLogs = logs.find(() => true);
-  const out = firstLogs?.events
+export function findAttribute(events: readonly Event[], eventType: string, attrKey: string): Attribute {
+  const out = events
```

Comment:
Let's keep this function unchanged since it is public. Instead we can make a new private one in cosmwasm-stargate to do the job we need.